x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/CrossClusterAsyncQueryIT.java

```diff
@@ -375,11 +375,9 @@ public void testStopQueryLocal() throws Exception {
         ActionFuture<EsqlQueryResponse> stopAction = client().execute(EsqlAsyncStopAction.INSTANCE, stopRequest);
         // ensure stop operation is running
         assertBusy(() -> {
-            try (EsqlQueryResponse asyncResponse = getAsyncResponse(client(), asyncExecutionId)) {
-                EsqlExecutionInfo executionInfo = asyncResponse.getExecutionInfo();
-                assertNotNull(executionInfo);
-                assertThat(executionInfo.isPartial(), is(true));
-            }
+            List<TaskInfo> tasks = getDriverTasks(client(REMOTE_CLUSTER_2));
```

Comment:
@dnhatn This part doesn't make sense to me - the purpose of the check has been to see if the "stop" operation has started, not that the cluster is finished, as in previous ones.